### PR TITLE
feat:add configuration for schema_locations, policy_config in config

### PR DESCRIPTION
**Pull request by Meyazhagan**
related issue - #737

### CODE REVIEW

**Comment by adifayer** [comment]
Thank you for raising this PR! @Meyazhagan We will look into it in the upcoming days 🤩

File: pkg/localConfig/localConfig.go [hunk 1]
```diff
@@ -101,14 +108,29 @@ func (lc *LocalConfigClient) Set(key string, value string) error {
 		return err
 	}
 
-	viper.Set(key, value)
+	tranformedValue := transformValue(key, value)
```
Comment (niv1612) [inline pkg/localConfig/localConfig.go]: The name `transformValue` isn’t indicative. I would suggest simply writing an inline if-else statement, something like this:
```
if key == policyConfigKey {
viper.Set(....)
} else if ....
```

File: cmd/test/main.go [hunk 2]
```diff
@@ -293,13 +298,22 @@ func GenerateTestCommandData(testCommandFlags *TestCommandFlags, localConfigCont
 
 	var policies *defaultPolicies.EvaluationPrerunPolicies
 	var err error
+	policyConfigEnv := os.Getenv(DatreePolicyConfig)
```
Comment (niv1612) [inline cmd/test/main.go]: Can you please explain why the environment variable is being used in this context? I’m not quite sure of its purpose and would appreciate some clarification.

File: cmd/test/main.go [hunk 3]
```diff
@@ -293,13 +298,22 @@ func GenerateTestCommandData(testCommandFlags *TestCommandFlags, localConfigCont
...
 
-	if testCommandFlags.PolicyConfig != "" {
+	if testCommandFlags.PolicyConfig != "" || localConfigContent.PolicyConfig != "" || policyConfigEnv != "" {
 		if localConfigContent.Offline != "local" && !evaluationPrerunDataResp.IsPolicyAsCodeMode {
-			return nil, fmt.Errorf("to use --policy-config flag you must first enable policy-as-code mode: https://hub.datree.io/policy-as-code")
+			return nil, fmt.Errorf("to use custom policy-config you must first enable policy-as-code mode: https://hub.datree.io/policy-as-code")
+		}
+
+		policyConfig := testCommandFlags.PolicyConfig
+		if policyConfig == "" {
```
Comment (niv1612) [inline cmd/test/main.go]: I noticed that there are two identical if statements. Was it made by mistake? It appears to be a potential bug

File: cmd/test/main.go [hunk 4]
```diff
@@ -317,14 +331,23 @@ func GenerateTestCommandData(testCommandFlags *TestCommandFlags, localConfigCont
 		return nil, err
 	}
 
+	schemaLocations := testCommandFlags.SchemaLocations
+	if len(schemaLocations) == 0 {
```
Comment (niv1612) [inline cmd/test/main.go]: Same here (two identical if statements). Was it made by mistake? If not, could you explain?

**Review by niv1612 — CHANGES_REQUESTED** [review]

File: cmd/test/main.go (hunk 3 above)
Comment (Meyazhagan) [reply cmd/test/main.go]: Not sure why I went with this approach. I was looking for `??` in go lang, wonder go was not support. May be that influenced.

I rewrote these conditions. Take a look at.

File: pkg/localConfig/localConfig.go (hunk 1 above)
Comment (Meyazhagan) [reply pkg/localConfig/localConfig.go]: I too love this simple way. I changed this to inline if-else

File: cmd/test/main.go (hunk 2 above)
Comment (niv1612) [reply cmd/test/main.go]: @Meyazhagan  I see that you are still using the environment variable for policyConfig. Can you explain why? We don't use environment variables currently in this context.

Comment (Meyazhagan) [reply cmd/test/main.go]: @niv1612 the user described, the solution to set policyConfig, either by config or env. 

> Order for reading values-
CLI flag command > Env variable > Config file

I didn't find a place for env wrapper like we have for localConfig. So I used it directly here.

**Review by niv1612 — APPROVED** [review]

**Pull request merged**